core/src/main/scala/org/apache/spark/sql/SnappyContext.scala

```diff
@@ -1005,12 +1005,15 @@ object SnappyContext extends Logging {
 //          else SplitClusterMode(sc, url)
           else throw new SparkException(
           s"Invalid configuration parameter ${Property.Locators}. " +
-              s"Use paramater ${Property.SnappyConnection} for smart connector mode")
+              s"Use parameter ${Property.SnappyConnection} for smart connector mode")
       }.orElse(Property.McastPort.getOption(conf).collectFirst {
         case s if s.toInt > 0 =>
           val url = "mcast-port=" + s
           if (embedded) ExternalEmbeddedMode(sc, url)
-          else SplitClusterMode(sc, url)
+//          else SplitClusterMode(sc, url)
+          else throw new SparkException(
+            s"Invalid configuration. " +
```

Comment:
will be better to have parameter name in message: "Invalid configuration parameter mcast-port"